action.yml

```diff
@@ -10,3 +10,7 @@ inputs:
   token:
     description: 'Github Token'
     required: true
+  dry-run:
```

Comment:
`dry-run` should be renamed to `dryrun` to follow convention seen on other actions.